ArraySorting.cls

```diff
@@ -0,0 +1,96 @@
+public class ArraySorting{
+	public static void ascendingOrder(List<Integer> values){
+		List<Integer> integerArray = new List<Integer>(values);
+		for(Integer i = 0 ; i < integerArray.size() ; i ++) {
+			for(integer j = i+1 ; j <= integerArray.size() -1 ; j ++ ){
+  				integer temporaryVariable = 0 ;
+  				if(integerArray[i] >  integerArray[j]){
+   					temporaryVariable = integerArray[i];
+   					integerArray[i] = integerArray[j];
+   					integerArray[j] = temporaryVariable;
+				}
+			}
+		}
+		System.debug(integerArray);
+	}
+	
+	public static void descendingOrder(List<Integer> elements){
+		List<Integer> arrayList = new List<Integer>(elements);
+		for(Integer i = 0 ; i < arrayList.size() ; i ++) {
+			for(integer j = i+1 ; j <= arrayList.size() -1 ; j ++ ){
+  				integer temporary_Var = 0 ;
+  				if(arrayList[i] <  arrayList[j]){
+   					temporary_Var = arrayList[i];
+   					arrayList[i] = arrayList[j];
+   					arrayList[j] = temporary_Var;			
+				}
+			}
+		}
+		System.debug(arrayList);
+	}
+	
+	
+	public static void maximumNumber(List<Integer> elementList){

```

Comment:
Here also you can merge `min` and `max` into one method
- [ ] Merge min and max into one method.